### ISSUE TITLE
OKEX RequestTimeout for all functions

### ISSUE THREAD

**Issue by salmonx** [ccxt/ccxt#2972]
- OS: Linux
- Programming Language version: python
- CCXT version: all
- Exchange: okex
- Method: all

OKEX RequestTimeout for all functions

**Comment by hezhongyu** [comment]
me too

**Comment by jackyxie** [comment]
v2 api has deprecated.

**Comment by kroitor** [comment]
They moved their products endpoint to a new URL. I'll fix it in 5-10 minutes.

**Comment by kroitor** [comment]
Fixed in CCXT 1.14.77. Please make sure to update your version. Thx for reporting!

**Comment by HornKild** [comment]
I updated `ccxt` with the latest version (1.14.79) to solve the timeout problem with OKEx but now I have an error loading objects. Uninstalling and reinstalling the package doesn't change anythig, am'I the only one facing this ? 

```
In [707]: ccxt.bittrex()
Traceback (most recent call last):

  File "<ipython-input-707-416616f1276a>", line 1, in <module>
    ccxt.bittrex()

  File "/home/bragar/anaconda3/lib/python3.6/site-packages/ccxt/base/exchange.py", line 219, in __init__
    settings = self.deep_extend(self.describe(), config)

  File "/home/bragar/anaconda3/lib/python3.6/site-packages/ccxt/bittrex.py", line 30, in describe
    return self.deep_extend(super(bittrex, self).describe(), {

TypeError: super(type, obj): obj must be an instance or subtype of type
```
